crates/agglayer-certificate-orchestrator/src/lib.rs

```diff
@@ -167,6 +228,21 @@ where
             return Poll::Ready(());
         }
 
+        // Poll the notification tasks to check for
+        match self.certifier_tasks.poll_join_next(cx) {
+            Poll::Ready(Some(Ok(Err(error)))) => {
+                error!("Error during p-proof generation: {:?}", error)
+            }
+            Poll::Ready(Some(Err(error))) => {
+                error!("Critical error during p-proof generation: {:?}", error);
+            }
+            Poll::Ready(Some(Ok(Ok(certifier_outputs)))) => {
+                info!("Received the successfully generated p-proof");
```

Comment:
Should it be `info` or `debug` ?

And also, can we link this log to a certificate ID?
```suggestion
                info!(certificate_id: certifier_outputs.certificate_id, "Received the successfully generated p-proof");
```

Reply:
I moved it to debug to align with the surrounding logs, as for adding certificate id to the context log, there is no such `certificate_id` field yet, probably adding the context log on top of the other PRs which reworked significantly the certificate data structure